subjects/corewar/README.md

```diff
@@ -0,0 +1,366 @@
+## Core War
+
+### Introduction
+
+Back in 1984 D. G. Jones had the brilliant idea to create a programming game inspired by how some viruses manage to take over a system and their internal memory.
+
+From the very start the game was a great success and a flourishing community added new features and organized tournaments worldwide.
+
+In this project we will recreate the necessary tools to run a more elaborate version of the game.
+
+### General instructions
+
+- The Virtual Machine and the Assembler can be done in any compiled language.
+- You should only use the standard library of the language you choose.
+- In case you use any external library it must be largely justified and explained (avoid it at any cost).
+- You must not use external libraries to do the parsing part of the Assembler for any reason.
+- The programs should never crash for any reason.
+- The programs should never leak memory for any reason.
+
+### Game dynamics
+
+At its very core the game consists of having two or more programs (written by the contestants) that will be executed by the Virtual Machine in an arena (a sandbox memory space).
```

Comment:
I would already use a term like `players` (or whatever we think it works best) instead of program